cmd/epithet-oidc-plugin/epithet-oidc-plugin.go

```diff
@@ -35,6 +43,28 @@ func main() {
 	}
 }
 
+func logging(cmd *cobra.Command, args []string) {
+	if logPath != "" {
+		f, err := os.OpenFile(logPath, os.O_WRONLY|os.O_APPEND|os.O_CREATE, 0644)
+		if err != nil {
+			// Disable file logging when there is an error
+			log.SetOutput(ioutil.Discard)
```

Comment:
I'd consider panic'ing here rather than quietly not logging

Reply:
Good call, fixed